Better explanation of "Reserved capacity type"

EricBergDE opened the pull request:
Changed the description of "Reserved capacity type" as this was not clear...

EricBergDE committed:
Better explanation of "Reserved capacity type"

Changed the description of "Reserved capacity type" as this was not clear...

SnehaGunda commented on articles/cosmos-db/cosmos-db-reserved-capacity.md:
```diff
@@ -50,7 +50,7 @@ The size of the reservation should be based on the total amount of throughput th
    |Subscription  |   Subscription that's used to pay for the Azure Cosmos DB reserved capacity. The payment method on the selected subscription is used in charging the upfront costs. The subscription type must be one of the following: <br/><br/>  Enterprise Agreement (offer numbers: MS-AZR-0017P or MS-AZR-0148P): For an Enterprise subscription, the charges are deducted from the enrollment's mone ...
    |Scope   |  	Option that controls how many subscriptions can use the billing benefit associated with the reservation. It also controls how the reservation is applied to specific subscriptions.   <br/><br/>  If you select **Single subscription**, the reservation discount is applied to Azure Cosmos DB instances in the selected subscription. <br/><br/>  If you select **Shared**, the reservation d ...
-   |Reserved capacity type   |  Throughput provisioned as request units.|
+   |Reserved capacity type   |  Throughput provisioned as request units. You can make reservations for to have all regions to process the writes (multi-master) or exactly one region to process the writes (single-master).|
```
I see a typo - For to (remove for)

EricBergDE committed:
Changed  description according recommendation

Changed to: You can buy a reservation for the provisioned throughput for both setups - single region writes as well as multiple region writes.
